docs/contribute/release_process.rst

```diff
@@ -86,20 +86,30 @@ The last step is to update the KEYS file with your code signing key https://www.
 Cut a Release Candidate
 -----------------------
 
-To cut a release candidate, one needs to first cut a branch using selected version string, e.g.,
+To cut a release candidate, one needs to first cut a branch using selected version string. Branches should be named with the base release version without the patch. For example, to cut a candidate for ``v0.11.0``, the branch should be ``v0.11`` and a tag named ``v0.11.0.rc0`` pushed to the HEAD of that branch once cut.
 
 .. code-block:: bash
 
 	git clone https://github.com/apache/tvm.git
 	cd tvm/
-	# Replace v0.6.0 with the relevant version
-	git branch v0.6.0
-	git push --set-upstream origin v0.6.0
 
-(*Make sure the version numbers in the source code are correct.* Run ``python3 version.py`` to update the version.)
+	# Update version numbers
+	# ...
+	git add .
+	git commit -m"Bump version numbers to v0.6.0"
+
+	# Replace v0.6 with the relevant version
+	git branch v0.6
+	git push --set-upstream origin v0.6
+
+	git tag v0.6.0.rc0
+	git push origin refs/tags/v0.6.0.rc0
+
+Make sure the version numbers in the source code are correct (example: https://github.com/apache/tvm/pull/14300). Run ``python3 version.py`` to update the version. Version numbers should be updated immediately after a release candidate branch is pushed.
```

Comment:
Should we add a note that PRs need to be marked as GPG "Verified" in GitHub, to be acceptable? I needed to jump quite a few hoops to get that and it was not obvious beforehand, so perhaps we could make that clear for the future Release Mgrs?